src/pytest_ansible/plugin.py

```diff
@@ -17,8 +20,12 @@
 )
 from pytest_ansible.host_manager import get_host_manager
 
+from .molecule import MoleculeFile
```

Comment:
since this import will pull in molecule and molecule may not be installed we can do something like this

```
try:
  from .molecule import Molecule
except ImportError:
  HAS_MOLECULE = False
```
and use the constant below